riocli/static_route/model.py

```diff
@@ -29,7 +29,6 @@ def find_object(self, client: Client) -> bool:
             'kind': 'staticroute',
             'nameOrGUID': self.metadata.name
```

Comment:
The code in this file should use the v2 static route APIs instead of the v1 APIs.